extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -99,22 +99,10 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     $('.reply_message_iframe_container:visible').last().append(this.factory.embeddedReply(params, false, true)); // xss-safe-value
   }
 
-  public scrollToElement = (selector: string) => {
```

Comment:
This removal isn't really related to the PR, but I decided to include it here because

1. it was only used to scroll to the secured reply box
2. the secured reply box is autofocused and therefore browsers will scroll to is natively
3. when the secured reply box was higher than the viewport this method was scrolling to the bottom and the cursor wasn't visible

Reply:
I originally added this because at least one browser did not scroll down at all. You'd click on reply button and if the reply box was below screen, you would not see anything happen. It would seem like it's broken.

I did notice that the scrolling is not perfect at times, let's try removing it as you did, and hopefully the old issues don't resurface.